### PR TITLE
[frontend/backend] Add triggers management in Organizations overview settings screen (#3520)

### PR DESCRIPTION
**Pull request by marieflorescontact**
### Proposed changes

* Add triggers management in Organizations overview settings screen
* Add test cases for organizations
* test counter:
  * +1 organization 
  * +1 relation (user => organization)
  * +1 event (organization creation )

### Related issues

* [issues/3520](https://github.com/OpenCTI-Platform/opencti/issues/3520)

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [x] I wrote test cases for the relevant uses case
- [x] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

this is the continuation and last part of issue/3520

### CODE REVIEW

**Comment by helene-nguyen** [comment]
[Frontend]

- The button is cut

Screenshot 2023-07-05 084018

- The display is the same even if deletion is enabled.

Screenshot 2023-07-05 090210

- Translation is missing

Screenshot 2023-07-05 084318
Screenshot 2023-07-05 091121

**Comment by helene-nguyen** [comment]
Locally tested, looks good!

**Comment by marieflorescontact** [comment]
> [Frontend]
> 
> * The button is cut
> 
> Screenshot 2023-07-05 084018
> 
> * The display is the same even if deletion is enabled.
> 
> Screenshot 2023-07-05 090210
> 
> * Translation is missing
> 
> Screenshot 2023-07-05 084318 Screenshot 2023-07-05 091121

Fixed

**Comment by marieflorescontact** [comment]
PR status : all comments have been taken into account, ready for more reviews / waiting for approval

**Comment by SarahBocognano** [comment]
- When modifying an entity with my user (part of organization X), I have a notification for the modification but I also have a notification for a suppression. I don't know why this comportement occurs.
- It's a shame that we are not notify when a relation between entities is created (entities that are selected in my trigger)
- If the admin modify an entity, the user doesn't have a notification of it (I don't know if it's the comportement that we attend)
- If the user delete an entity, I have no notification of it (I've noticed that when we select "creation", "modification" and "suppression", it has trouble to be well executed. Like sometimes it works, sometimes it doesn't. 

Capture d’écran (48)
For this trigger, I have selected "creation", "modification" and "suppression". Could have been nice to see them directly in the component (I saw that they are present in the DOM but hidden)

File: opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js
Comment (SarahBocognano) [inline opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js]: Wow that's a big change ! Can you explain that ?

**Review by SarahBocognano — COMMENTED** [review]
Tested 👍🏼

File: opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js
Comment (marieflorescontact) [reply opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js]: oups..
fixed

**Comment by marieflorescontact** [comment]
PR status : Fixing test counter, ready for more reviews / waiting for approval

**Comment by marieflorescontact** [comment]
> For this trigger, I have selected "creation", "modification" and "suppression". Could have been nice to see them directly in the component (I saw that they are present in the DOM but hidden)

 You are right but as discuss with @Kedae it needs to be discuss

**Comment by marieflorescontact** [comment]
PR status : Ready for more reviews / waiting for approval

**Review by Archidoit — APPROVED** [review]

**Pull request merged**
